### DISCUSSION TITLE
Same template for all specific taxonomy terms

### DISCUSSION THREAD

**Discussion by megasupernoob** [statamic/cms#4975] [Q&A]
I have a Statamic (3.2.x) site with these taxonomies:

TAXONOMIES:
-- Categories
----- cat 1
----- cat 2
----- cat 3
-- Tags
------ tag 1
------ tag 2
... etc.

Currently if I create a new Category > Term, for example, "cat 4", I also need to create a template for it. What I want to do is to use a single template for all Categories, and a single template for all Tags. So if I or another site administrator creates a new category term then it would automatically use the default Categories antler template.

How can I achieve that?
Thanks!

**Comment by edalzell** [comment]
Why do you think you need a template for each term? Have you seen the docs on [views](https://statamic.dev/taxonomies#views)?
